proto/vectorindex.proto

```diff
@@ -37,9 +37,17 @@ message _Vector {
 
 message _Metadata {
     string field = 1;
+
+    message ListOfStrings {
+        repeated string values = 1;
+    }
+
     oneof value {
-        // Eventually can support ints, dates, etc
         string string_value = 2;
+        int64 integer_value = 3;
+        double double_value = 4;
+        bool boolean_value = 5;
+        ListOfStrings list_of_strings_value = 6;
```

Comment:
I had this instead `_ListOfStrings list_of_strings_value = 6;`

Following what other message naming

Reply:
I can change that, though it seems to be inconsistent in this file, eg
![image](https://github.com/momentohq/client-protos/assets/3690240/d2bea366-15b2-4641-958c-dfcdb751008b)

Reply:
Fixed in https://github.com/momentohq/client-protos/pull/226/commits/f88b92d6cc4b5bfc8a1f257fa71d817035b57b5c